functions/java-jre.bats

```diff
@@ -1,73 +1,76 @@
 #!/usr/bin/env bats
 
-load java-jre
-load helpers
-
-@test "installation-java_exist" {
-# TODO: rewrite test, in CI it fails because java is not installed at this point
-skip
-  run java -version
-  if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
-  [ "$status" -eq 0 ]
-  [[ $output == *"Zulu"* ]]
-}
+load java-jre.bash
+load helpers.bash
 
 @test "destructive-install_zulu8-64bit" {
-  echo -e "# \e[36mZulu 8 64-bit Java installation is being (test-)installed..." >&3
+  echo -e "# ${COL_CYAN}$(timestamp) [openHABian] Zulu 8 64-bit Java installation is being (test-)installed...${COL_DEF}" >&3
   case "$(uname -m)" in
     aarch64|arm64|x86_64|amd64) ;;
     *) skip ;;
   esac
-  run java_zulu_fetch Zulu8-64
+  run java_zulu_prerequsite "Zulu8-64" 3>&-
```

Comment:
what's that trailing minus ?

Reply:
Its to close the BATS fd 3 stream as described [here](https://github.com/bats-core/bats-core#file-descriptor-3-read-this-if-bats-hangs) because I was having issues with BATS hanging.

Reply:
ok